packages/storybook-webpack-angular-types-plugin/src/lib/webpack-angular-types-plugin/plugin.ts

```diff
@@ -56,23 +60,118 @@ export class WebpackAngularTypesPlugin {
 				smallTsProject.resolveSourceFileDependencies();
 
 				for (const { path, module } of modulesToProcess) {
-					const { classesInformation, interfacesInformation } = generateTypeInformation(
+					const {
+						classesInformation,
+						interfacesInformation,
+						functionsInformation,
+						constantsInformation,
+					} = generateTypeInformation(
 						path,
 						smallTsProject,
 						this.options.excludeProperties,
 					);
-					for (const ci of classesInformation) {
-						this.addClassCodeDocDependency(ci, module);
+					for (const classInformation of classesInformation) {
+						this.addClassCodeDocDependency(classInformation, module);
 					}
-					for (const ii of interfacesInformation) {
-						this.addInterfaceCodeDocDependency(ii, module);
+					for (const interfaceInformation of interfacesInformation) {
+						this.addInterfaceCodeDocDependency(interfaceInformation, module);
+					}
+
+					const groupedExportsInformation = this.groupExportInformation(
+						functionsInformation,
+						constantsInformation,
+					);
+
+					for (const groupedExportInformation of groupedExportsInformation) {
+						this.addGroupedExportsCodeDocDependency(groupedExportInformation, module);
 					}
```

Comment:
So this only works on a module level? So you can not group functions etc. across modules?

Reply:
That's a mistake, haven't thought about that! Good catch. I will change the logic that functions and constants are first collected across all modules and then grouped.